### ISSUE TITLE
Error building docker cpu image (boost package changed)

### ISSUE THREAD

**Issue by dgtlmoon** [beniz/deepdetect#186]
Looks like something has changed in boost library and so deepdetect can no longer be built when running `docker build .` from `deepdetect/docker/cpu`

```
CMake Error at /usr/share/cmake-2.8/Modules/FindBoost.cmake:1131 (message):
  Unable to find the requested Boost libraries.

  Boost version: 1.54.0

  Boost include path: /usr/include

  Could not find the following Boost libraries:

          boost_iostreams

  Some (but not all) of the required Boost libraries were found.  You may
  need to install these additional Boost libraries.  Alternatively, set
  BOOST_LIBRARYDIR to the directory containing Boost libraries or BOOST_ROOT
  to the location of Boost.
Call Stack (most recent call first):
  CMakeLists.txt:53 (find_package)


```

**Comment by dgtlmoon** [comment]
Full build log
[log.txt](https://github.com/beniz/deepdetect/files/477722/log.txt)

**Comment by beniz** [comment]
Thanks for spotting this one. Fixed by commit above, due to new #176.
